spec/address_book_spec.rb

```diff
@@ -190,4 +192,53 @@ def check_entry(entry, expected_name, expected_number, expected_email)
       expect(entry).to be_nil
     end
   end
+
+  context '#iterative_search' do
+    it 'searches AddressBook for a non-existent entry' do
+      book.import_from_csv('entries.csv')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.